project/devs.sls

```diff
@@ -32,3 +32,16 @@ include:
 
 {% endfor %}
 {% endif %}
+
+{% if 'users' in pillar and pillar['users'] %}
+purge_users:
+  cmd.script:
+    - cwd: /var/www/{{ pillar['project_name']}}
+    - name: salt://users/purge-users.py
+    - args: {% for user in pillar['users'] %}--keep={{ user }} {% endfor %}
+    - user: root
+    - require:
+{% for user in pillar['users'] %}
+      - user: {{ user }}
+{% endfor %}
```

Comment:
I want to be sure all the current users are configured correctly before removing old users, in case anything goes bad in between.